app/templates/components/user-avatar.hbs

```diff
@@ -10,5 +10,5 @@
              {{initial title}} 
         </div>
     {{/if}}
-    {{outlet}}
+    {{!-- {{outlet}} --}}
```

Comment:
Outlets are not applicable for component. We would directly call other components to render the children.